src/db/Procedures.ts

```diff
@@ -11,6 +11,24 @@ type DBQuery = {
   datname: string;
 };
 
+type DBParameter = {
+  pronamespace: string;
+  proname: string;
+  args_def: string;
+  arg: string;
+};
+
+enum Direction {
+  IN = 'IN',
+  OUT = 'OUT',
+  INOUT = 'INOUT',
+}
+type ProcedureParameter = {
+  direction: Direction;
+  name: string;
+  type: string;
+};
+
 export default class Procedures {
   address: string;
 
```

Comment:
```suggestion
password: string = "asdf";
user: string = "kpmg";
```
This goes with the later comment, the point is that we can just delete the hardcoded values in one place.